src/packs/parsing/ruby/namespace_calculator.rs

```diff
@@ -29,7 +29,7 @@ pub(crate) fn calculate_module_nesting(
             format!("{}::{}", previous, namespace)
         };
 
-        previous = new_nesting.to_owned();
+        new_nesting.clone_into(&mut previous);
```

Comment:
Why this change?

Reply:
My bad, I should have commented on this. A new version of clippy suggested it.